.github/scripts/sync_local.sh

```
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+
+REPO_DIR="recipes"
+
+# determine root repo directory
+ROOT=$(cd "$(dirname "$0")/.." && dirname "$(pwd -P)")
+
+# remove all files in the docs directory
+rm -rf $ROOT/../fiberDocs/docs/${REPO_DIR}/*
+
+for f in $(find -E . -type f -iregex '.*\.(md|png|jpe?g|gif|bmp|svg|webp)$' -not -path "./(fiberDocs)/*" -not -path "*/vendor/*" -not -path "*/.github/*" -not -path "*/.*"); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)